trillian/migrillian/core/controller.go

```diff
@@ -228,30 +228,72 @@ func (c *Controller) runWithRestarts(ctx context.Context) error {
 // have been transferred (in non-Continuous mode).
 func (c *Controller) Run(ctx context.Context) error {
 	metrics.controllerStarts.Inc(c.label)
+	dur := randDuration(c.opts.StopAfter, c.opts.StopAfter)
```

Comment:
What is 'dur' here the total run time allowed? Renaming it might help. It's hard to tell if the 2nd stopAfter is what's really needed.

Reply:
Renamed to `stopAfter`. Yes, it's the total run time allowed, after which mastership for this log will be rotated.

The second `StopAfter` is the max amount of jitter added to the first `StopAfter`, so we stop after rand `[StopAfter, 2*StopAfter)`. Could be a separate parameter, but I just keep it simpler for now. This PR doesn't change the formula, by the way.

Reply:
Yeah could maybe use stopAfter/2 so it won't run longer than the config syas but I don't think the exact formula is too critical.

Reply:
Good idea. I may do it in a follow-up PR, because it would change the current behavior.